### ISSUE TITLE
About

### ISSUE THREAD

**Issue by FritzdorfSport** [frksteenhoff/ClimApp#148]
A few people get name-checked in Preferences-About, but none of the "core" team.

It might be quite nice to have a list of the team with their affiliations, where it says "This App is made possible by all the contributions from the ClimApp team:
Boris Kingma  affiliation
Chuansi Gao affiliation
Lars Nybo xxxxx

and so on.

**Comment by Chuansi-LU** [comment]
Good suggestion. We can use the name "ClimApp Project consortium: …...". The work was supported financially by the ClimApp project, which is a part of ERA4CS, an ERA-NET initiated by JPI Climate, and funded by FORMAS (SE, grant 2017-01739), IFD (DK, case 7110-00003B), NWO (NL, grant 438.17.806) with co-funding by the European Union (Grant 690462), http://www.jpi-climate.eu/nl/25223441-ClimApp.html

**Comment by BorisKingma** [comment]
fair point - will at least link to the member page of ClimApp - (so we only have 1 source to maintain)
